### ISSUE TITLE
[ISSUE] databricks_model_serving resource failing to update

### ISSUE THREAD

**Issue by tiagoshin** [databricks/terraform-provider-databricks#3593]
We have an issue when we try to update a databricks_model_serving resource. 
Deploying for the first time is ok, it creates the Endpoint properly. But, when we need to run the terraform apply again, it says that we're changing the workload type and size when we aren't changing it. In fact, we aren't even defining these configurations.

### Configuration
```hcl
resource "databricks_model_serving" "chat_model" {
  provider = databricks.workspace
  name     = "${var.environment}-${var.service}-chat-model"
  config {
    served_entities {
      name                       = "${var.environment}-${var.service}-chat-model"
      entity_name                = "databricks_meta_llama_3_models.models.meta_llama_3_8b_instruct"
      entity_version             = "1"
      scale_to_zero_enabled      = var.databricks_scale_to_zero_enabled
      min_provisioned_throughput = var.databricks_min_provisioned_throughput
      max_provisioned_throughput = var.databricks_max_provisioned_throughput
    }
    traffic_config {
      routes {
        served_model_name  = "${var.environment}-${var.service}-chat-model"
        traffic_percentage = 100
      }
    }
  }
}
```

### Expected Behavior
The expected behavior is that it doesn't identify changes when there aren't any changes.

### Actual Behavior
```hcl
  # databricks_model_serving.chat_model will be updated in-place
  ~ resource "databricks_model_serving" "chat_model" {
        id                  = "name of the resource"
        name                = "name of the resource"
        # (1 unchanged attribute hidden)
      ~ config {
          ~ served_entities {
                name                       = "name of the resource"
              ~ scale_to_zero_enabled      = false -> true
              - workload_size              = "Small" -> null
              ~ workload_type              = "MULTIGPU_MEDIUM" -> "CPU"
                # (5 unchanged attributes hidden)
            }
            # (1 unchanged block hidden)
        }
    }
```
And we got error:
```
Error: cannot update model serving: failed to reach NOT_UPDATING, got UPDATE_FAILED: current status: UPDATE_FAILED
```

### Steps to Reproduce
1. Create the resource
2. Try to run terraform plan and apply again, with no changes.

### Terraform and provider versions
Databricks provider version 1.44.0

### Is it a regression?
No, we used the most updated

### Important Factoids
Yes, the Databricks team enabled some hardware on this workspace. Please check with @Ahmed Bilal

### Would you like to implement a fix?
No

**Comment by alexott** [comment]
@tiagoshin, can you provide sample values for all variables? Also, your TF template doesn't have any value for `workload_type`, `workload_size`, etc.

**Comment by tiagoshin** [comment]
Sure, the variables are like:
- var.environment = "dev"
- var.service = "service-name"
- var.databricks_scale_to_zero_enabled = "true"
- var.databricks_min_provisioned_throughput = 3600
- var.databricks_max_provisioned_throughput = 7200

> Also, your TF template doesn't have any value for workload_type, workload_size, etc.
That's the point! The new Databricks model serving model is not based on workload size and type. For e.g., we cannot even define them on UI. Instead, we use `min_provisioned_throughput` and `max_provisioned_throughput`, so the hardware is assigned by Databricks. 
So, if I'm not defining those values, why does Terraform want to change them?

**Comment by alexott** [comment]
@tiagoshin 

Can you attach logs as per first section of [troubleshooting guide](https://registry.terraform.io/providers/databricks/databricks/latest/docs/guides/troubleshooting)?

Regarding `workload_type` and `workload_size` - it looks like they are set on the backend, so they are populated in the TF state, and when apply happens, then TF figures out that they aren't set, and tries to reset them.  To avoid resetting of that values they need to be correctly marked in the code as `computed`. Or they should be not populated by the backend...

**Comment by tiagoshin** [comment]
@alexott How do we mark them as `computed`?

**Comment by clintbxtreme** [comment]
To test I also tried defining `workload_type` and `workload_size` but terraform is still wanting to change the `scale_to_zero_enabled` flag even tho it was previously applied initially: 
```# databricks_model_serving.chat_model will be updated in-place
  ~ resource "databricks_model_serving" "chat_model" {
        id                  = "name of the resource"
        name                = "name of the resource"
        # (1 unchanged attribute hidden)
      ~ config {
          ~ served_entities {
                name                       = "name of the resource"
              ~ scale_to_zero_enabled      = false -> true
                # (7 unchanged attributes hidden)
            }
            # (1 unchanged block hidden)
        }
    }
```
This also resulted in the same error as above

**Comment by alexott** [comment]
Fixed by #3608
